### PR TITLE
Card styling

### PR DESCRIPTION
**Pull request by Kyruel87**
Re-styled both main cards and dog card.

### CODE REVIEW

**Comment by lambley** [comment]
The curved borders look a bit off - if you want to fix it, can you try using custom classes rather than bootstrap classes?

Also capitalize the restaurant type on the card
image

**Review by lambley — CHANGES_REQUESTED** [review]
see my other comment for changes

**Comment by Kyruel87** [comment]
> The curved borders look a bit off - if you want to fix it, can you try using custom classes rather than bootstrap classes?
> 
> Also capitalize the restaurant type on the card image

I left the restaurant type lower case as it was before because I thought that was someone's styling choice 😅. Will work on these now

**Comment by lambley** [comment]
Still not quite right sorry - see screenshot.
- you can see the white background behind the curved border
- image crop and radius is leading to unintended results - images shouldn't be this zoomed in. 

Please fix before I can approve this. 

image

File: app/views/shared/_card.html.erb [hunk 1]
```diff
@@ -3,22 +3,24 @@
     <div class="row g-0">
       <div class="col-md-4">
 
-        <%= cl_image_tag(activity.cl_tag, width: 150, height: 150, class: "card-position") %>
+        <%= cl_image_tag(activity.cl_tag, :format => :png, class: "card-img", transformation:[ {width: 245, height: 200, crop: "crop", gravity:"auto", quality: "auto"},
```
Comment (lambley) [inline app/views/shared/_card.html.erb]: this needs to be fixed - the transformation here is leading to an odd zoom in effect

File: app/views/shared/_card.html.erb [hunk 2]
```diff
@@ -3,22 +3,24 @@
...
+{radius: "25:0:0:25"} ] ) %>
```
Comment (lambley) [inline app/views/shared/_card.html.erb]: also this is on a new line for some reason

**Comment by lambley** [comment]
finally, i would recommend using your own classes and not overwriting bootstrap classes, as this leads to unintended affects: https://getbootstrap.com/docs/4.0/components/card/

File: app/views/shared/_card.html.erb (hunk 1 above)
Comment (Kyruel87) [reply app/views/shared/_card.html.erb]: It's using AI to select the best part of the image to crop to as most of the images aren't very shrinkable. Played around quite a bit with them getting some to look nice as others were grainy or some other sort of artifacts. Will work it out

**Review by lambley — COMMENTED** [review]
ok this is good. had to adjust the css slightly to make card heights uniform, but will merge now.

**Comment by lambley** [comment]
I've tried my best to resolve these conflicts and I can't unfortunately. There's a mix of custom and bootstrap classes that make it difficult to fix. Sorry but I will close this and redo the changes.

**Comment by Kyruel87** [comment]
What was it doing? I can take out the bootstrap classes and fix it when I
get to class, I wonder why they would work fine on my end though

**Pull request closed without merging**